x-pack/legacy/plugins/canvas/i18n/components.ts

```diff
@@ -13,4 +14,38 @@ export const ComponentStrings = {
         defaultMessage: 'No matching objects found.',
       }),
   },
+  HelpMenu: {
+    getHelpMenuDescription: () =>
+      i18n.translate('xpack.canvas.helpMenu.description', {
+        defaultMessage: 'For {CANVAS} specific information',
+        values: {
+          CANVAS,
+        },
+      }),
+    getDocumentationLinkLabel: () =>
+      i18n.translate('xpack.canvas.helpMenu.documentationLinkLabel', {
+        defaultMessage: '{CANVAS} documentation',
+        values: {
+          CANVAS,
+        },
+      }),
+    getKeyboardShortcutsLinkLabel: () =>
+      i18n.translate('xpack.canvas.helpMenu.keyboardShortcutsLinkLabel', {
+        defaultMessage: 'Keyboard Shortcuts',
+      }),
+  },
+  KeyboardShortcutsDoc: {
+    getTitle: () =>
+      i18n.translate('xpack.canvas.keyboardShortcutsDoc.flyoutHeaderTitle', {
+        defaultMessage: 'Keyboard Shortcuts',
+      }),
+    getShortcutSeparator: () =>
+      i18n.translate('xpack.canvas.keyboardShortcutsDoc.shortcutListSeparator', {
+        defaultMessage: 'or',
```

Comment:
I'm pretty sure we can't do this, but I might be wrong.  At the very least, it needs `description` added.

Translators need to see these strings in context.  This word might translate into several in another language that don't make sense, (in this case, several shortcuts and an alternative).

I would reach out to the i18n wranglers to be sure this can be translated.

@bmcconaghy , can you lend some insight here?

Reply:
Yeah just a bare "or" will not really work as translators will not know how it is being used in context.  As Clint suggests, this approach:
```
defaultValue: "{shortcut1}, {shortcut2} or {shortcut3}"
values: {...}
```
is preferable.